code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubTokenHandler.java

```diff
@@ -47,20 +48,16 @@
     private GithubClient client;
 
     private static final String GITHUB_REQUEST_TOKEN = "code";
-    private static final String GITHUB_USER_ACCOUNT_KIND = "user";
-    private static final String GITHUB_ADMIN_ACCOUNT_KIND = "admin";
-    private static final String GITHUB_EXTERNAL_TYPE = "github";
 
     private static final DynamicLongProperty TOKEN_EXPIRY_MILLIS = ArchaiusUtil.getLong("api.auth.jwt.token.expiry");
     private static final DynamicBooleanProperty SECURITY = ArchaiusUtil.getBoolean("api.security.enabled");
-    private static final DynamicStringProperty WHITELISTED_ORGS = ArchaiusUtil.getString("api.auth.github.allowed.orgs");
-    private static final DynamicStringProperty WHITELISTED_USERS = ArchaiusUtil.getString("api.auth.github.allowed.users");
-    private static final DynamicStringProperty ACCESS_MODE = ArchaiusUtil.getString("api.auth.github.access.mode");
 
     @Inject
     ProjectResourceManager projectResourceManager;
     @Inject
     ObjectManager objectManager;
+    @Inject
+    GithubUtils githubUtils;
```

Comment:
I believe that this is not static because it has a few classes internally it uses that are injected and as a result needs to be a bean. I could be wrong i can rename it if you would like to something like GithubHelper